### PR TITLE
Improve the OTP flow states

### PR DESCRIPTION
**Pull request by ahmed-gz**
## Description

[YouTrack issue](https://todo.irdcorp.dev/issue/SID-000)

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have generated the new version of the docs website and smoke tested it
- [ ] I have checked that my changes haven't caused semantic errors in the existing docs
- [x] I have generated a `changeset` if my change affects the published packages

### CODE REVIEW

File: .changeset/olive-onions-prove.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@slashid/react": patch
+---
+
+Improve the OTP flow states
```
Comment (ikovic) [inline .changeset/olive-onions-prove.md:5]: This message will end up in the automatically generated changelog - we need to be a bit more specific than this. Can we list the changes we did here?

File: packages/react/src/domain/handles.ts [hunk 2]
```diff
@@ -112,7 +122,8 @@ export type AuthenticatingMessage = {
   message: TextConfigKey;
 };
 export function getAuthenticatingMessage(
```
Comment (ikovic) [inline packages/react/src/domain/handles.ts:124]: Can we check if this function is being used anywhere else but in `authenticating.tsx`? If not, I would suggest moving it to the same file so it has a stronger context.

Comment (ahmed-gz) [reply packages/react/src/domain/handles.ts:124]: You're right, it's only used in in `authenticating.tsx` I will move it there

File: .changeset/olive-onions-prove.md (hunk 1 above)
Comment (ahmed-gz) [reply .changeset/olive-onions-prove.md:5]: Sorry you're right, I will add more details

**Review by ikovic — APPROVED** [review]
Thanks @ahmed-gz, this looks good!

**Pull request merged**